tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -205,11 +211,17 @@ def collect_results(self, sleep_time_secs):
         return data
 
     @cluster(num_nodes=7)
+    @matrix(failure_mode=["clean_shutdown", "hard_shutdown", "clean_bounce", "hard_bounce"],
+            broker_type=["leader"],
+            num_threads=[1, 3],
+            sleep_time_secs=[120],
+            metadata_quorum=[quorum.remote_kraft])
     @matrix(failure_mode=["clean_shutdown", "hard_shutdown", "clean_bounce", "hard_bounce"],
             broker_type=["leader", "controller"],
             num_threads=[1, 3],
             sleep_time_secs=[120])
```

Comment:
I added this extra matrix configuration to avoid `broker_type=="controller"` for `metadata_quorum=="quorum.remote_kraft"`-- not sure if you noticed that, otherwise I would have just added `metadata_quorum=quorum.all_non_upgrade` to the existing matrix to cover both modes as you said.  This is because killing the controller when it's separate from the broker is maybe not what the test is attempting to test and also requires other changes to accommodate killing off the separate KRaft controller node, but that seems a bit overkill for this purpose.

Is that what

Reply:
Thanks, @AlanConfluent . I was actually making a slightly different suggestion, namely to explicitly include `metadata_quorum=[quorum.zk]` in this second matrix config, and to remove the default parameter value `metadata_quorum=quorum.zk` from the method signature.

Github charmingly won't let me do a suggestion to make this clearer because the diff includes a deleted line.

Reply:
I see.  Done!